ivy/functional/ivy/layers.py

```diff
@@ -1217,12 +1217,12 @@ def depthwise_conv2d(
 def conv3d(
     x: Union[ivy.Array, ivy.NativeArray],
     filters: Union[ivy.Array, ivy.NativeArray],
```

Comment:
Similarly here

Reply:
yeah it is noted in the docstring that the function is nestable and also accepts container classes as noted in the guide

Reply:
Can you add them here as inputs as well for completeness?

Reply:
done